components/canvas/webgl_thread.rs

```diff
@@ -3297,11 +3323,13 @@ impl Drop for WebXRBridgeManager {
     }
 }
 
+#[cfg(feature = "webxr")]
 struct WebXRBridgeContexts<'a> {
     contexts: &'a mut FnvHashMap<WebGLContextId, GLContextData>,
     bound_context_id: &'a mut Option<WebGLContextId>,
 }
 
+#[cfg(feature = "webxr")]
```

Comment:
My instinct is that it would be more clear to extract as many of these webxr-specific structures into a separate module that can have a single cfg applied to it, which will reduce the number of one-off cfgs needed in this file.